wren-ai-service/src/pipelines/generation/followup_sql_generation.py

```
@@ -78,6 +86,7 @@ def prompt(
     instructions: list[dict] | None = None,
     has_calculated_field: bool = False,
     has_metric: bool = False,
+    sql_functions: List[SqlFunction] | None = None,
```

Ruff (0.8.2)

Undefined name `List`

(F821)